src/text-editor.coffee

```diff
@@ -677,7 +677,7 @@ class TextEditor extends Model
   # this editor.
   shouldPromptToSave: ({windowCloseRequested}={}) ->
     if windowCloseRequested
-      @isModified()
+      false
```

Comment:
When would these buffers be deserialized? Would it only be when atom is opened in the same project context? i.e. What would happen in this case:

* open atom via `atom /somepath`
* edit untitled buffers
* kill atom 
* open atom via `atom /anotherpath`

Will the scratch buffers be in this new atom window? Or only if I started it via `atom /somepath`?